src/test/data/ManualTesting/[T12-B3][People.connect]TestScript.adoc

```diff
@@ -0,0 +1,329 @@
+```
+
+---
+`edit 1 e/rowanA@example.com` +
+```
+Edited Person: Rowan Atkinson Phone: 9234-1666 Email: rowanA@example.com Address: 132, Beverly Hills, #02-145 Birthday: 01/12/1975 Photo: BradPitt.jpg Tags: [celebrity]
+```
+
+---
+`edit 1 a/426, Ang Mo Kio Ave 3, #11-123` +
+```
+Edited Person: Rowan Atkinson Phone: 9234-1666 Email: rowanA@example.com Address: 426, Ang Mo Kio Ave 3, #11-123 Birthday: 01/12/1975 Photo: BradPitt.jpg Tags: [celebrity]
+```
+
+---
+`edit 1 b/06/01/1955` +
+```
+Edited Person: Rowan Atkinson Phone: 9234-1666 Email: rowanA@example.com Address: 426, Ang Mo Kio Ave 3, #11-123 Birthday: 06/01/1955 Photo: BradPitt.jpg Tags: [celebrity]
+```
+
+---
+`edit 1 pt/RowanAtkinsonNew.jpg` +
+```
+Edited Person: Rowan Atkinson Phone: 9234-1666 Email: rowanA@example.com Address: 426, Ang Mo Kio Ave 3, #11-123 Birthday: 06/01/1955 Photo: RowanAtkinsonNew.jpg Tags: [celebrity]
+```
+
+---
+`edit 1 t/celebrity t/friends` +
+```
+Edited Person: Rowan Atkinson Phone: 9234-1666 Email: rowanA@example.com Address: 426, Ang Mo Kio Ave 3, #11-123 Birthday: 06/01/1955 Photo: RowanAtkinsonNew.jpg Tags: [celebrity][friends]
+```
+
+=== Delete
+
+`delete` +
+```
+Invalid command format!
+delete: Deletes the person identified by the index number used in the last person listing.
+Parameters: INDEX (must be a positive integer)
+Example: delete 1
+```
+
+---
+`delete John` +
+```
+Invalid command format!
+delete: Deletes the person identified by the index number used in the last person listing.
+Parameters: INDEX (must be a positive integer)
+Example: delete 1
+```
+
+---
+`delete 11` +
+```
+The person index provided is invalid
+```
+
+---
+`delete 10` +
+```
+Deleted Person: John Doe Phone: 9004-2345 Email: johnd@example.com Address: 468A, Admiralty Drive, #06-127 Birthday: 19/12/1999 Photo: johnD.png Tags: [friends]
+```
+
+=== Load
+
+`load` +
+```
+Invalid command format!
+load: Loads contacts from a pre-existing address book to the current one. The pre-existing address book's name is given as a parameter.
+Parameters: FILENAME
+Example: load myaddressbook.xml
+```
+
+---
+`load newAddressbook.txt` +
+```
+The address book couldn't be read. Make sure your file is in the right directory and that it's in the correct format.
+```
+
+---
+`load newAddressbook.xml` +
+```
+Successfully loaded the address book.
+```
+
+=== Find
+
+`find` +
+```
+Invalid command format!
+find: Finds all persons whose names contain any of the specified keywords (case-sensitive) and displays them as a list with index numbers.
+Parameters: KEYWORD [MORE_KEYWORDS]...
+Example: find alice bob charlie
+```
+
+---
+`find 1` +
+```
+0 persons listed!
+```
+
+---
+`list`
+```
+Listed all persons
+```
+
+---
+`find John` +
+```
+1 persons listed!
+```
+
+---
+`list`
+```
+Listed all persons
+```
+
+=== List by tags
+
+---
+`list`
+```
+Listed all persons
+```
+
+---
+`list t/1` +
+```
+No person with given tags found.
+```
+
+---
+`list t/friends` +
+```
+Listed all persons
```

Comment:
A better test will be to use a command which finds only few people and not lists all the people since it will list all people anyway without the tag argument.